### PR TITLE
Update pom file

### PR DESCRIPTION
**Pull request by 2136146qwe**

### CODE REVIEW

File: src/main/java/managers/AppManager.java [hunk 1]
```diff
@@ -19,23 +19,31 @@ public class AppManager {
...
 
-    public static WebDriver getWebDriver(){return driver;}
-    public UserHelper getUserHelper(){return userHelper;}
-    public NavigationHelper getNavigationHelper(){return navigationHelper;}
-}
\ No newline at end of file
+    public static WebDriver getWebDriver() {
+        return driver;
+    }
+
```
Comment (pprettytony) [inline src/main/java/managers/AppManager.java:41]: remove empty line

File: src/main/java/pages/HomePage.java [hunk 2]
```diff
@@ -14,36 +15,20 @@ public HomePage(PageManager pages) {
...
 
-    @FindBy(xpath = "/html/body/app-root/div/div/rz-header/rz-main-header/header/div/div/ul/li[3]/rz-user/button/svg")
-    WebElement viewProfileMenuButton;
+    @FindBy(xpath = "//button[@class='header__button ng-star-inserted']")
+    WebElement userMenuButton;
 
-    public void clickOnTheSignInButton() {
-        wait.until(ExpectedConditions.elementToBeClickable(signInButton)).click();
-    }
+    @FindBy(xpath = "//button[@class='auth-modal__register-link button button--link ng-star-inserted']")
```
Comment (pprettytony) [inline src/main/java/pages/HomePage.java]: check how to use xpath contains()

File: src/main/java/managers/AppManager.java (hunk 1 above)
Comment (2136146qwe) [reply src/main/java/managers/AppManager.java:41]: done

File: src/main/java/pages/HomePage.java (hunk 2 above)
Comment (2136146qwe) [reply src/main/java/pages/HomePage.java]: done

**Pull request merged**